ruby/extractor/Cargo.toml

```diff
@@ -9,12 +9,12 @@ edition = "2018"
 [dependencies]
 flate2 = "1.0"
 node-types = { path = "../node-types" }
-tree-sitter = "0.19"
+tree-sitter = "0.20"
 tree-sitter-embedded-template = "0.19"
 tree-sitter-ruby = { git = "https://github.com/tree-sitter/tree-sitter-ruby.git", rev = "bb6a42e42b048627a74a127d3e0184c1eef01de9" }
-clap = "2.33"
+clap = "3.0"
```

Comment:
The two `Cargo.toml` files have conflicts, could you resolve these please. In addition, the [Cargo.lock](https://github.com/github/codeql/blob/ef227a47219aef17c62bfe881b6459ef20ceb828/ruby/Cargo.lock) file needs to be updated to reflect the dependency changes.